pkgs/by-name/ak/akarin-expr/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  llvm_18,
+  pkg-config,
+  cmake,
+  vapoursynth,
+  ninja,
+  libxml2,
+}:
+
+stdenv.mkDerivation {
+  name = "akarin-expr";
+  version = "0.95";
+
+  src = fetchFromGitHub {
+    owner = "Jaded-Encoding-Thaumaturgy";
+    repo = "vapoursynth-plugin";
+    rev = "6d7c733b3014a42be75299427b5c35f56f02a47a";
+    hash = "sha256-MlQ0iOYPMAGttOVD2yF4kyZE+cpQ240DC2TBJY8/pH4=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    pkg-config
+    cmake
+    ninja
+  ];
+  buildInputs = [
+    llvm_18
+    vapoursynth
+    libxml2
+  ];
+
+  postPatch = ''
+    substituteInPlace meson.build \
+      --replace-fail \
+        "install_dir: join_paths(vapoursynth_dep.get_pkgconfig_variable('libdir'), 'vapoursynth')," \
+        "install_dir: join_paths('${placeholder "out"}/lib/', 'vapoursynth'),"
+  '';
+
+  meta = {
+    description = ""; # I have no clue what the description should be :<
```

Comment:
This probably needs to be addressed. Maybe `An experimental VapourSynth plugin` would do?

Reply:
Yup, I'll jus drop the `An` article